installer/PowerToysSetup/Product.wxs

```diff
@@ -368,23 +368,16 @@
     <DirectoryRef Id="FancyZonesInstallFolder" FileSource="$(var.BinX64Dir)modules\">
       <Component Id="Module_FancyZones" Guid="C6B5272E-6ED4-4B80-B0E7-2FF0355D8CF4" Win64="yes">
         <File Source="$(var.BinX64Dir)modules\$(var.FancyZonesProjectName)\fancyzones.dll" KeyPath="yes" />
+        <File Source="$(var.BinX64Dir)modules\$(var.FancyZonesProjectName)\FancyZonesEditor.dll" />
+        <File Source="$(var.BinX64Dir)modules\$(var.FancyZonesProjectName)\FancyZonesEditor.runtimeconfig.json" />
         <File Source="$(var.BinX64Dir)modules\$(var.FancyZonesProjectName)\FancyZonesEditor.exe" >
           <netfx:NativeImage Id="FancyZonesEditor.exe" Platform="64bit" Priority="0" />
```

Comment:
This won’t work on .NET Core, as (a) `ngen.exe` is not supported there, and (b) this file will no longer have any C# code in it (it is a C++-only bootstrapper tool). You’ll need to get rid of this line.